### DISCUSSION TITLE
github actions should show runner version

### DISCUSSION THREAD

**Discussion by malytomas** [community/community#127810] [Actions]
### Select Topic Area

Product Feedback

### Body

My tests for a c++ project run on github actions started failing in some configurations. However, running the tests on a commit that previously succeeded now fails too. This tells me that the issue is related to some change in github actions hosted runners. This is extremely difficult for me to debug as I have no information about what has changed in the environment.

I propose that github actions should always log the version of the hosted runner. Preferably with a link to the particular commit in https://github.com/actions/runner.

**Comment by Boomtrexx** [comment]
hello

**Comment by malytomas** [comment]
found it :facepalm:
